vertical-pod-autoscaler/README.md

```diff
@@ -240,35 +243,39 @@ spec:
 
 To diagnose problems with a VPA installation, perform the following steps:
 
-* Check if all system components are running:
-```
+- Check if all system components are running:
+
+```console
 kubectl --namespace=kube-system get pods|grep vpa
 ```
+
 The above command should list 3 pods (recommender, updater and admission-controller)
 all in state Running.
 
-* Check if the system components log any errors.
+- Check if the system components log any errors.
   For each of the pods returned by the previous command do:
-```
+
+```console
 kubectl --namespace=kube-system logs [pod name] | grep -e '^E[0-9]\{4\}'
 ```
 
-* Check that the VPA Custom Resource Definition was created:
-```
+- Check that the VPA Custom Resource Definition was created:
+
+```console
 kubectl get customresourcedefinition | grep verticalpodautoscalers
 ```
 
 ### Components of VPA
 
 The project consists of 3 components:
 
-* [Recommender](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/recommender/README.md) - it monitors the current and past resource consumption and, based on it,
+- [Recommender](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/recommender/README.md) - it monitors the current and past resource consumption and, based on it,
```

Comment:
Can we do:
[Recommender](https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/recommender/README.md) - monitors the current and past resource consumption and, based on it,

Reply:
Good idea, fixed in 586813cbb5